Source/Core/Editor/UI/ProcessSceneObjectEditor.cs

```diff
@@ -99,6 +99,11 @@ private void SetupGroupManagement(VisualElement root)
 
             List<IGuidContainer> groupContainers = targets.Where(t => t is IGuidContainer).Cast<IGuidContainer>().ToList();
 
+            if (AdvancedSettings.Instance.ShowExpertInfo == false)
+            {
+                objectIdLabel.RemoveFromHierarchy();
+            }
+
             DisplayObjectGuid(objectIdLabel);
```

Comment:
Why do we do DisplayObjectGuid this with AdvancedSettings.Instance.ShowExpertInfo == false

Reply:
Basically so the code looks tidier, but you're right it's not needed, I put it into an else block.